data/test_e2ebot/config.yml

```diff
@@ -15,3 +15,4 @@ policies:
 - name: TEDPolicy
   random_seed: 42
   epochs: 5
+- name: RulePolicy
```

Comment:
added it to test fingerprinting when one of the policies doesn't need e2e features